File: module_common/src/main/java/com/callhh/module_common/bean/BaseResponseBean.java
package com.callhh.module_common.bean;

import java.io.Serializable;

/**
 * 基础http响应实体
 */
public class BaseResponseBean implements Serializable {

    private static final long serialVersionUID = -1477609349345966116L;

    public int code;
    public String msg;

    public ResponseBean toResponseBean() {
        ResponseBean responseBean = new ResponseBean();
        responseBean.code = code;
        responseBean.msg = msg;
        return responseBean;
    }
}

File: library_abtool/src/main/java/com/callhh/abtool/bean/BaseResponseBean.kt
package com.callhh.abtool.bean

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * 基础http响应实体
 */
@Parcelize
open class BaseResponseBean : Parcelable {

    var code = 0
    var msg: String = ""

    fun toResponseBean(): ResponseBean<*> {
        val responseBean: ResponseBean<*> = ResponseBean<Any>()
        responseBean.code = code
        responseBean.msg = msg
        return responseBean
    }

}